front/tests/pages/simulation-conf-page.ts

```diff
@@ -24,7 +24,7 @@ class SimulationConfPage extends RollingStockSelectorPage {
     this.pathfindingResultDistance = page.getByTestId('result-pathfinding-distance');
     this.infraLoadingState = page.locator('.infra-loading-state');
     this.pathfindingState = page.locator('.pathfinding-state-main-container');
-    this.seachByTrigramRocketButton = page.getByTestId('rocket-button');
+    this.searchByTrigramRocketButton = page.getByTestId('rocket-button');
```

Comment:
In `operational-studies-page-model.ts`, it's called `searchByTrigramButton`. Should we keep harmoniozed names when targeting the same button ?

Reply:
Will be handled in this PR #9306